### PR TITLE
i18n: update zh-cn translation

### LINKED ISSUE #6326: [i18n] Translation Chinese (简体中文 zh-cn) 🇨🇳 needs translators
Dear FreshRSS-Community,

please help us to improve our Chinese (zh-cn) translations.

While the core developer and many other contributors develop our [FreshRSS ](https://freshrss.org/) app we cannot support the translations in all 20+ languages. 

## Actual Status
Right now about 90% of our text is translated into Chinese (zh-cn).

## How to support us
How to do it:
1) it is very simple
2) all files are stored in `https://github.com/FreshRSS/FreshRSS/tree/edge/app/i18n/zh-cn` ([Link](https://github.com/FreshRSS/FreshRSS/tree/edge/app/i18n/zh-cn))
3) each untranslated text has an inline comment `//  TODO`
4) When the text is translated, just delete the comment
5) If the English term fits as well, please change the comment from `//  TODO` to `//  IGNORE`
6) open a Pull Request and link to this ticket

## Thanks
Thanks a lot. It will help our very international community of users and developers. Your submitted PRs will be published with our next release version [1.24.0](https://github.com/FreshRSS/FreshRSS/milestones) or later

### PR DESCRIPTION
**Pull request by yzqzss**
closes #6326

### CODE REVIEW

File: app/i18n/zh-cn/conf.php [hunk 1]
```diff
@@ -16,8 +16,8 @@
 		'exception' => '清理例外',
 		'help' => '更多可用选项位于各订阅源的设置',
 		'keep_favourites' => '永不删除已收藏的文章',
-		'keep_labels' => '永不删除标签',
-		'keep_max' => '最多保留的文章数',	// DIRTY
+		'keep_labels' => '永不删除打了标签的文章',
+		'keep_max' => '每订阅源最多保留的文章数',
```
Comment (yzqzss) [inline app/i18n/zh-cn/conf.php:20]: follow #5458

File: app/i18n/zh-cn/sub.php [hunk 2]
```diff
@@ -127,46 +127,46 @@
 				'xpath' => 'XPath 定位：',
 			),
 			'json_dotpath' => array(
-				'_' => 'JSON (Dotted paths)',	// TODO
+				'_' => 'JSON (点表达式)',
```
Comment (yzqzss) [inline app/i18n/zh-cn/sub.php:130]: Have translated "Dotted paths" as "dot-notation" and so on according to #6317.

File: app/i18n/zh-cn/sub.php [hunk 3]
```diff
@@ -178,10 +178,10 @@
 		'max_http_redir' => '最大 HTTP 重定向',
 		'max_http_redir_help' => '设置为 0 或留空以禁用，-1 表示无限重定向',
 		'method' => array(
-			'_' => 'HTTP Method',	// TODO
+			'_' => 'HTTP 方式',
 		),
-		'method_help' => 'The POST payload has automatic support for <code>application/x-www-form-urlencoded</code> and <code>application/json</code>',	// TODO
-		'method_postparams' => 'Payload for POST',	// TODO
+		'method_help' => 'POST 荷载自动支持 <code>application/x-www-form-urlencoded</code> 和 <code>application/json</code>',
+		'method_postparams' => 'POST 荷载',
```
Comment (yzqzss) [inline app/i18n/zh-cn/sub.php:184]: One other thing. Perhaps `method_postparams` should be renamed to `method_postpayload`?

Comment (Frenzie) [reply app/i18n/zh-cn/sub.php:184]: Params is clearer than payload imo (unless there are no multiple params involved but only a single payload).

Comment (yzqzss) [reply app/i18n/zh-cn/sub.php:184]: https://github.com/FreshRSS/FreshRSS/blob/9c97d8ca729e3cfb067445c0d3c9ad8284132aeb/app/views/subscription/add.phtml#L333
https://github.com/FreshRSS/FreshRSS/blob/350edf398c55b472e19a3017de9b4d2d3420b9e4/app/Controllers/feedController.php#L184
https://github.com/FreshRSS/FreshRSS/blob/350edf398c55b472e19a3017de9b4d2d3420b9e4/app/Controllers/feedController.php#L209

How about `method_postfields`?

Comment (Frenzie) [reply app/i18n/zh-cn/sub.php:184]: That's fine by me. :-)

**Comment by math-GH** [comment]
Right now we have 2 translations.... see #6335 too

**Comment by math-GH** [comment]
I fixed the conflict, because https://github.com/FreshRSS/FreshRSS/pull/6317 renamed `dot paths` => `dot notation`.

**Comment by yzqzss** [comment]
> Right now we have 2 translations.... see #6335 too

It's OK for me to rebase and create a new PR after #6335 is merged.

Or, @nanhualyq would you be willing to cherry-pick some of my translations? feel free to do that! I think some of my translations are a little more accurate :), I'll close mine if you cherry-pick them.

**Comment by nanhualyq** [comment]
> > Right now we have 2 translations.... see #6335 too
> 
> It's OK for me to rebase and create a new PR after #6335 is merged.
> 
> Or, @nanhualyq would you be willing to cherry-pick some of my translations? feel free to do that! I think some of my translations are a little more accurate :), I'll close my if you cherry-pick them.

Yours is better, so close mine.

**Comment by yzqzss** [comment]
> > > Right now we have 2 translations.... see #6335 too
> > 
> > 
> > It's OK for me to rebase and create a new PR after #6335 is merged.
> > Or, @nanhualyq would you be willing to cherry-pick some of my translations? feel free to do that! I think some of my translations are a little more accurate :), I'll close my if you cherry-pick them.
> 
> Yours is better, so close mine.

ah, Thank you.

---

@FreshRSS could you please add @nanhualyq as a co-author when merging?  

`Co-authored-by: nanhualyq <nanhualyq@gmail.com>`

93e275870d47f8a02e5c323eaeace04229a1d8fd I forgot to add username, so GitHub didn't recognize it.

**Pull request merged**